NewSpring/crete
# feat: Adds Multiple Campaign Items Functionality

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR makes it so that you can have multiple items in the campaign content channel and it will pick the most recent non-future dated one to show.

### How do I test this PR?
There are 3 items in the campaign channel in `alpha-rock`. Two are in the past, one is in the future. The one that shows up in the campaign item in the app should be the most recent one that isn't in the future. 

Note: if you have `ROCK_SHOW_INACTIVE_CONTENT` set to `true`, you will see the one in the future instead. You should set this setting to `false` for this test.

Note #2: There is also a change in here that brings over the `getCursorBySiblingContentItemId` function. This is because we needed to change this so that the horizontal content items didn't show campaign items. 

## TODO


## REVIEW


---

> The purpose of a PR Review is to _improve the quality of the software._
